plugins/quetz_repodata_zchunk/tests/test_main.py

```diff
@@ -142,7 +142,7 @@ def test_repodata_zchunk(
             with open('repodata.json') as f:
                 repodata_unzck = f.read()
 
-            assert repodata == repodata_unzck  # NOQA
+            assert repodata == repodata_unzck  # NOQA # type: ignore
```

Comment:
This seems utterly broken. The `repodata` variable isn't defined anywhere. Do we run the plugin tests in CI?

Reply:
We do, i can check this tomorrow 👍🏼

Reply:
Probably the `if` branch is never run, and the `else` branch doesn't contain any asserts 🤷🏼‍♂️